src/components/app/app.scss

```diff
@@ -1,28 +1,48 @@
 @import '../../styles/variables';
 
+//-- Theme colours as CSS Custom Properties --//
+
+.kui-theme--light {
+  --color-default: #{$color-light};
```

Comment:
idk, I guess it could, but I don't think it'd be simpler to read or understand. My inclination is not to over-optimise here.

Reply:
Okay fine for now, but I think it's worth automating this eventually for ease of updating colours and to avoid them both potentially diverging accidentally as they're very easy to mix up.

Reply:
Sure. Can you give me an example of how you'd do it? I've not come up with any ideas that are simpler than the status quo. Everything I can think of just feels like a cumbersome abstraction that doesn't actually simplify things much